examples/filter_before_line_number.rs

```diff
@@ -5,19 +5,27 @@ use std::{
 
 use libc::c_char;
 use tree_sitter::Node;
+use tree_sitter_grep::{
+    PluginInitializeReturn, PLUGIN_INITIALIZE_ARGUMENT_NOT_PARSEABLE,
+    PLUGIN_INITIALIZE_MISSING_EXPECTED_ARGUMENT, PLUGIN_INITIALIZE_SUCCEEDED,
```

Comment:
So this obviously wants to be an enum but I didn't know how/if you could use a Rust enum as a C-FFI type? So went for the moment with "how maybe you're supposed do it" via a simple type alias for `u32` + named constants for the different expected error/success codes?

Reply:
Ok updated to use a `#[repr(u8)]` enum instead

So if someone wanted to write a plugin in something other than Rust then they wouldn't be able to import the Rust types anyway (so that didn't make sense really)?

So they would just follow documentation that said "return 0 for success, 1 for missing argument and 2 for unparseable argument (where the returned type is the C equivalent of `u8` (`unsigned char`?))"?

Reply:
Yeah, makes sense